lib/product.service.ts

```diff
@@ -33,14 +39,59 @@ export class ProductStack extends Stack {
       }
     });
 
+    // DynamoDB tables
+    const productsTable = new dynamo.Table(this, 'ProductsTable', {
+      partitionKey: { name: 'id', type: dynamo.AttributeType.STRING },
+     });
 
+    const stockTable = new dynamo.Table(this, 'StockTable', {
+      partitionKey: { name: 'product_id', type: dynamo.AttributeType.STRING },
+    });
+
+    // Table names created on asw console
+    const PRODUCTS_TABLE_NAME = 'Products';
+    const STOCK_TABLE_NAME = 'Stock';
+
+    // DynamoDB lambdas
+    const getProductsListLambda = new NodejsFunction(this, 'get-products-list-lambda', {
+      functionName: 'getProductsList',
+      entry: path.resolve(__dirname, '../src/products/products-dynamodb.handler.ts'),
+      runtime: lambda.Runtime.NODEJS_18_X,
```

Comment:
there are some 3-4 common props set to these lambdas here. We can create the common  const for that and reuse in other places
```
const commonLambdaProps = { 
       runtime: lambda.Runtime.NODEJS_18_X,
      environment: {
        PRODUCTS_TABLE_NAME: PRODUCTS_TABLE_NAME,
        STOCK_TABLE_NAME: STOCK_TABLE_NAME
      }
      ...
  }
```